testsuite/features/step_definitions/salt_steps.rb

```diff
@@ -500,6 +500,12 @@
   find(:xpath, "//div[@id='#{sectionids[section]}']/button").click
 end
 
+When(/^I press minus sign in (.*) section$/) do |section|
+  sectionids = { 'third configured zone' => 'bind#configured_zones#2',
```

Comment:
I think it would be very practical to have that `sectionids` inside our map `FIELD_IDS` in `constants.rb`

Reply:
Feel free to finish moving things there in some other PR.

BTW there's still quite a lot in other steps.

Reply:
I agree with Oscar here. I started to move the code in functions like this one: https://github.com/aplanas/uyuni/commit/d81169b1f73ea1264c066961b89045411680d312